lib/2_uppercases_all_letters.rb

```diff
@@ -5,5 +5,7 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def uppercases_all_letters(str)
-  # Write your code here.
+    if str
```

Comment:
I don't have a good answer, being that I didn't tell the function to do anything else, and it will probably keep returning an error without the "str" variable being defined. I will repush this one.